src/main/java/com/rackspace/ceres/app/services/DataTablesStatements.java

```diff
@@ -29,6 +30,7 @@
  * configuration-driven data tables schema.
  */
 @Component
+@Slf4j
```

Comment:
Where are we using this in here?

Reply:
I missed to remove that as I was using it for local debugging.